frontend/src/app/components/complete-sprint/complete-sprint.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { CompleteSprintDialogComponent } from './complete-sprint-dialog/complete-sprint-dialog.component';
+
+@Component({
+  selector: 'app-complete-sprint',
+  templateUrl: './complete-sprint.component.html',
+  styleUrls: ['./complete-sprint.component.sass']
+})
+export class CompleteSprintComponent implements OnInit {
+
+  public completeBthClass = 'fill gray';
+
+  constructor(public matDialog: MatDialog) { }
+
+  ngOnInit(): void {
+  }
+
+  OpenCompSprintDialog(): void {
```

Comment:
for method name should be used lowerCamelCase

Reply:
I fixed it